steps/property-inspection/alterations.tsx

```diff
@@ -64,6 +66,33 @@ const step: ProcessStepDefinition<ProcessDatabaseSchema, "property"> = {
         value: "Save and continue",
       }),
     componentWrappers: [
+      ComponentWrapper.wrapStatic<ProcessDatabaseSchema, "property">(
+        new StaticComponent({
+          key: "paragraph-1",
+          Component: Paragraph,
+          props: {
+            children: `Explain the alterations procedure and that written permission is required before any alterations or improvements cannot be carried out.`,
+          },
+        })
+      ),
+      ComponentWrapper.wrapStatic<ProcessDatabaseSchema, "property">(
+        new StaticComponent({
+          key: "paragraph-2",
+          Component: Paragraph,
+          props: {
+            children: (
+              //TODO add correct url
+              <>
+                Information about alterations is in{" "}
+                <Link href={""} target="_blank">
+                  [Your Tenancy conditions: section 4.19]
+                </Link>{" "}
```

Comment:
What should the `href` value for this link be? Do we know it?

Also, Balsamiq adds square brackets around text to show that they are  links. Can you remove them here please?

Reply:
```suggestion
                <Link href={""} target="_blank">
                  Your Tenancy conditions: section 4.19
                </Link>{" "}
```